packages/hub/schema/field.js

```diff
@@ -216,15 +216,10 @@ module.exports = class Field {
     return errors;
   }
 
-  customAnalyzer() {
-    if (typeof this.plugin.customAnalyzer === 'function') {
-      return Object.assign({}, this.plugin.customAnalyzer());
-    }
-  }
 
   mapping(searchTree, allFields) {
     let m = {
-      [this.id]: Object.assign({}, this.plugin.defaultMapping(allFields))
+      [this.id]: Object.assign({}, allFields)
     };
```

Comment:
This whole `mapping` function can get removed. And look for whatever is calling it (probably in content-type.js) and prune that back too.